src/views/settings/filters/parts/filter-tabs.tsx

```diff
@@ -51,7 +51,7 @@ const FilterTabs: FC = (): ReactElement => {
 		],
 		[]
 	);
-	const onItemClick = useCallback((selectedId) => {
+	const onChange = useCallback((selectedId) => {
```

Comment:
I'm sorry but this still does not work. The BC changes also the implementation of the onChange to
`onChange: (ev: SyntheticEvent<Element, Event>, selectedId: string) => void`

You can type the useCallback like this `useCallback<TabBarProps['onChange']>` to have a complete type checking, otherwise your selectedId param is an implicit any

Reply:
ok thanks, i'm on it